frontend/src/content/news.json

```diff
@@ -1,5 +1,10 @@
 {
   "2021": [
+    {
+      "date": "2021-12-06",
+      "title": "Metabolic Atlas 2.2 fixes bugs",
```

Comment:
Maybe change to "fixes critical bugs"?

Reply:
It's not my preferred choice of words but sure I can go for that.

Reply:
Applying this via a suggestion:
```suggestion
      "title": "Metabolic Atlas 2.2 fixes critical bugs",
```